.github/workflows/reusable-build.yml

```diff
@@ -20,6 +20,16 @@ on:
         required: false
         type: string
         default: Tuesday
+      build_stable_daily:
```

Comment:
This is fine for now, but it feels a bit like a belt and suspenders situation. We are being overly specific it feels like.